sources/models/src/vmware-k8s-1.19/defaults.d/50-vmware-k8s.toml

```diff
@@ -0,0 +1,71 @@
+[configuration-files.containerd-config-toml]
```

Comment:
This feels like a lot of duplication to me.  I think we could separate 50-aws-k8s.toml into configs for containerd, kubernetes-services, kubernetes-settings, and metrics, and then all of kubernetes-settings (the biggest piece) and metrics would be the same, with just the small kubernetes-settings and containerd configs changing...

Reply:
I like the idea, but I'd rather not make those larger changes as part of this PR.

Reply:
What if I do it as a separate PR merged before this one?

Reply:
With https://github.com/bottlerocket-os/bottlerocket/pull/1542 you should be able to link to the shared kubernetes-containerd.toml, kubernetes-metrics.toml, kubernetes-network.toml, and kubernetes-services.toml files, and creating a new shared kubernetes-vmware.toml file with the bits that remain.

Reply:
#1542 is merged, hopefully this would be easy now :)